wafer/management/static.py

```diff
@@ -19,10 +15,3 @@ def render_path(self, path=None, view=None):
                                                                      view)
             except IOError as err:
                 print('Skiping %s - threw IOError %s' % (path, err))
-            # This is a hack because dajngo_medusa doens't understand 301
```

Comment:
This hack exists because django-medusa converts every response that isn't 200 into a fatal generic Exception.

As long as we have the RedirectView in pages/urls.py, I'm pretty sure we're going to need something to handle this case, since we'll still be encountering a redirect while walking the tree of possible urls.